test/cluster/multicluster_test.go

```diff
@@ -16,27 +18,39 @@ var _ = Describe(
 
 		Context(
 			"When DeployMultiClusterMultiNs", func() {
-				// 1st cluster
-				clusterName1 := "multicluster"
-				clusterNamespacedName1 := getNamespacedName(
-					clusterName1, test.MultiClusterNs1,
-				)
-
-				// 2nd cluster
-				clusterName2 := "multicluster"
-				clusterNamespacedName2 := getNamespacedName(
-					clusterName2, test.MultiClusterNs2,
-				)
-
```

Comment:
Was there any issue in using the these Context level vars? 
We are using them as read-only

Reply:
no issues.

Reply:
used ginkgo process id